Neos.ContentRepository.Core/Classes/Feature/NodeModification/Dto/SerializedPropertyValues.php

```diff
@@ -43,7 +43,7 @@ public static function createEmpty(): self
     }
 
     /**
-     * @param array<string,mixed> $propertyValues
+     * @param array<string,array{type:string,value:mixed}|SerializedPropertyValue|null> $propertyValues
```

Comment:
Still weird to me. Passing `SerializedProperty` would make PHP complain, as the type is declared as `array`. 🤔

Reply:
The `SerializedPropertyValue` type hint is within the `array<...>` so it is an array of `string`, `array`, `SerializedPropertyValue` or `null`

Reply:
Yes this is "hip" syntax of phpstan/psalm/phpstorm https://phpstan.org/writing-php-code/phpdoc-types#array-shapes